docs/development.md

```diff
@@ -0,0 +1,106 @@
+# Development setup
+Make sure all of the applications and their dependencies have been installed correctly, configure the correct parameters
+and follow the installation instructions for each application.
+
+Accounts can be added to the `$config` in your git-ignored `simplesamlphp/config/authsources.php` file. 
+For development purposes, it could be useful to add the SRAA, RAA and RA accounts:
+
+```php
+    ...
+        'sraa:sraa' => array(
+            'commonName'             => 'SRAA',                 // displayName
+            'mail'                    => 'sraa@sraa.example',   // email (also used for EPTI)
+            'schacHomeOrganization'   => 'Example Inc',         // schacHomeOrganisation
+        ),
+        'raa:raa' => array(
+            'commonName'             => 'RAA',                  // displayName
+            'mail'                    => 'raa@raa.example',     // email (also used for EPTI)
+            'schacHomeOrganization'   => 'Example Inc',         // schacHomeOrganisation
+        ),
+        'ra:ra' => array(
+            'commonName'             => 'RA',                   // displayName
+            'mail'                    => 'ra@ra.example',       // email (also used for EPTI)
+            'schacHomeOrganization'   => 'Example Inc',         // schacHomeOrganisation
+        )
+    ...
+```
+
+To whitelist the organisation of these accounts, use the [Middleware Manager API][middleware-manager]:
+
+```
+    POST /app_dev.php/management/whitelist/add HTTP/1.1
+     Host: mw-dev.stepup.coin.surf.net
+     Accept: application/json
+     Content-Type: application/json
+     Authorization: Basic bWFuYWdlbWVudDpiYXI=
+     Cache-Control: no-cache
+
+     {
+       "institutions": [
+         "Example Inc"
+       ]
+     }
+```
+
+To allow account `sraa` to be added as an SRAA, use the [Middleware Manager API][middleware-manager] (request body has been truncated) to configure
+its NameID as such: 
+
+```
+      POST /app_dev.php/management/configuration HTTP/1.1
+      Host: mw-dev.stepup.coin.surf.net
+      Accept: application/json
+      Content-Type: application/json
+      Authorization: Basic bWFuYWdlbWVudDpiYXI=
+      Cache-Control: no-cache
+
+      {
+          "sraa": [
+            "9971dbcf01267b11f6107d9cafb43e5b4009a955"
+          ],
+          ...
+      }
+```
+
+To add account `sraa` by its NameID as an SRAA, run the following for Stepup-Middleware, replacing `<YOUR_YUBIKEY_NUMBER>` 
+with the number relating to your Yubikey (it is usually printed on it):
+
+```
+    app/console middleware:bootstrap:identity-with-yubikey 9971dbcf01267b11f6107d9cafb43e5b4009a955 'Example Inc' 'SRAA' sraa@sraa.example nl_NL <YOUR_YUBIKEY_NUMBER>
+```
+
+To add `raa` and `ra` as RAA and RA respectively, they first have to go through the vetting process.
+Their verification tokens can be registered by `sraa` in RA. The accounts can then also be added as an RAA or RA.
```

Comment:
Beetje ubiquitous language:

verification tokens == second factor tokens
registered == vetted
their identities can also be accredited as RA or RAA